packages/studio-web/src/app/demo/demo.component.ts

```diff
@@ -189,6 +237,91 @@ export class DemoComponent implements OnDestroy, OnInit {
       element.click();
       document.body.removeChild(element);
       this.registerDownloadEvent();
+    } else if (this.selectedOutputFormat === "zip") {
+      let zipFile = new JSZip();
+      // Create inner folder
+      const innerFolder = zipFile.folder("readalong");
+      const assetsFolder = innerFolder?.folder("assets");
+      const timestamp = new Date()
+        .toISOString()
+        .replace(/[^0-9]/g, "")
+        .slice(0, -3);
+      const basename = `readalong-${timestamp}`;
```

Comment:
When people provide a title, adding the title to the filename will help them find it after download, especially if they do multiple books.
   const basename = (this.slots.title?this.slots.title.replace(" ","_").tolowerCase():'readalong')+`-${timestamp}`;